app/Http/Controllers/SetupDocumentSignatureController.php

```
@@ -0,0 +1,54 @@
+    }
+    public function store(StoreSetupDocumentSignatureRequest $request)
+    {
+        $signature = SetupDocumentSignature::create($request->validated());
+        return SetupDocumentSignatureResource::make($signature)
+            ->additional([
+            'success' => true,
+            'message' => 'Document signature created successfully.',
+            ])
+            ->response();
+    }
+    public function show(SetupDocumentSignature $setupDocumentSignature)
```

PHPMD (2.15.0)

Avoid excessively long variable names like $setupDocumentSignature. Keep variable name length under 20. (Naming Rules)

(LongVariable)